### DISCUSSION TITLE
Arduino Web Timers deployed. Interactive & visual, with both atmel328p and lgt8f328p support!

### DISCUSSION THREAD

**Discussion by dbuezas** [dbuezas/lgt8fx#172] [Show and tell]
# https://dbuezas.github.io/arduino-web-timers

<img width="847" alt="screenshot" src="https://user-images.githubusercontent.com/777196/123638819-e5726f80-d81f-11eb-9689-0f05124df734.png">

I'll be super thankful if the community gives it a ride and reports bugs and thoughts if found :) 
(please use the appropriate repo for that)
It took me quite a bit of effort to add all LGT timer features and simulate & visualise them correctly, it is ridiculous how many more features LGT has compared to the atmega.

Most of the features I didn't even know before I started making this web app.

**Comment by navr32** [comment]
Very good works. many thanks for this.
Have a nice day.

**Comment by LaZsolt** [comment]
The tool is finished. Look, [click here](https://github.com/dbuezas/lgt8fx/discussions/164#discussioncomment-4751048).
